src/styles/colors.styl

```diff
@@ -1,17 +1,21 @@
 $cerulean = #00aaed
+$cerulean-2 = #00a9f0
+$curious-blue = #1FA6DF
 $picton-blue = #20ADED
 $cornflower-blue = #4D90FE
 $night-rider = #333333
 $thunder = #363436
+$emperor = #504E50
 $scorpion = #605D60
+$ironside-gray = #646261
 $fedora = #6B686B
 $mountain-mist = #A1A0A2
 $silver = #C1C1C1
 $pale-slate = #c4c3c4
 $venetian-red = #D0021B
 $iron = #dde0e0
 $quartz = #DEDEDF
-$athens-grey = #f5f6f8
+$athens-gray = #f5f6f8
```

Comment:
-1 gray.  grey for life!

Reply:
I think the rule is "gray" in **A**merica and "grey" in **E**ngland. And "grehy" in Canad**eh**